unit-tests/func/MUR/l500/test-Reading-MUR-option-off.cpp

```diff
@@ -0,0 +1,41 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../func-common.h"
+
+// Test group description:
+//       * This tests group verifies depth sensor max usable range option restrictions,
+//         according to [RS-8358]
+//
+// Current test description:
+//       * Reading Max Usable Range while Max Usable Range option is off - expect exception thrown
+//       
+// Note: * L515 specific test
+//       * Test will pass if no L515 device is connected
+//       * Test will be performed on the first L515 device detected.
+
+TEST_CASE( "Reading Max Usable Range while option is off - Fail Test", "[max-usable-range]" )
```

Comment:
Let's start adding metadata to the tests (at the top of the file):
```
//#test:tag max-usable-range lidar
//#test:device L500*
//#test:device L500* USB2
```
?

Reply:
I can maybe see how the tags can be taken automatically from the `TEST_CASE` 2nd param, but I think it'd be better verbose

Reply:
I added metadata as discussed